docker-hub/builds/troubleshoot.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Troubleshoot your builds
+description: How to troubleshoot automated builds
+keywords: docker hub, troubleshoot, automated builds, autobuilds
+---
+
+> **Note**
+>
+> Automated builds requires a
```

Comment:
Same as previous comment.
The plural here is awkward to me. If it were a feature name, like "The Automated Builds feature requires ...", then it reads okay. As is, maybe change to "Automated builds require ..."